addons/website/views/snippets/s_showcase.xml

```diff
@@ -5,52 +5,41 @@
 <template name="Showcase" id="s_showcase">
     <section class="s_showcase pt48 pb48" data-vcss="002">
         <div class="container">
-            <div class="row g-0 s_col_no_resize s_col_no_bgcolor s_nb_column_fixed">
-                <div class="col-lg text-lg-end">
+            <div class="row">
+                <div class="col-lg-6">
                     <div class="row">
-                        <div class="col-lg-12 pt24 pb24" data-name="Block">
-                            <div class="s_showcase_title d-flex flex-lg-row-reverse align-items-center mb-2">
-                                <i class="s_showcase_icon fa fa-2x fa-desktop text-secondary me-3 me-lg-0 ms-lg-3"/>
-                                <h3>First feature</h3>
-                            </div>
-                            <p>A short description of this great feature.</p>
+                        <div class="col-12 pb32">
+                            <h3>Features showcase</h3>
+                            <p class="lead">A features section highlights your product’s key attributes, engaging visitors and boosting conversions. <a href="#">Discover all the features</a></p>
                         </div>
-                        <div class="col-lg-12 pt24 pb24" data-name="Block">
-                            <div class="s_showcase_title d-flex flex-lg-row-reverse align-items-center mb-2">
-                                <i class="s_showcase_icon fa fa-2x fa-paint-brush text-secondary me-3 me-lg-0 ms-lg-3"/>
-                                <h3>Second feature</h3>
+                        <div class="col-12 pt8 pb8" data-name="Feature">
+                            <i class="s_showcase_icon fa fa-star-o me-auto rounded float-start" role="img"/>
+                            <div class="px-2 overflow-hidden">
```

Comment:
This might work, but since the image alignment function also sets a `me-auto` or `ms-auto` depending on direction, I'd have to use an `! important` flag to override it 😬 .